### PR TITLE
SMP 0.23.0 RN Dec 4th 2028

### PR DESCRIPTION
**Pull request by nataraja2018**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

**Review by pratmit — APPROVED** [review]
LGTM

**Review by arya-harness — APPROVED** [review]

File: release-notes/self-managed-enterprise-edition.md [hunk 1]
```diff
@@ -220,6 +220,213 @@ To fix this issue, follow these steps
...
+   ```
+
+   Harness helm chart supports configuring HPA for either all services at once or for selective services.
+
+### Fixed issues
+
+#### Cloud Cost Management
+- Fixed ECS Metrics Chart Issue: CPU recommendations in ECS metrics charts previously displayed incorrect graphs, and CPU usage showed as 0. This has now been corrected. [CCM-19715]
+
+#### Code Repository
```
Comment (bharatgoelharness) [inline release-notes/self-managed-enterprise-edition.md:326]: why this change?

Comment (bharatgoelharness) [reply release-notes/self-managed-enterprise-edition.md:326]: code is not part of SMP

Comment (nataraja2018) [reply release-notes/self-managed-enterprise-edition.md:326]: It is added earlier releases also exists.. Will remove both of them
